css/mobile_first.css

```diff
@@ -0,0 +1,147 @@
+:root {
+  --blanco: #f9f9f9;
+  --negro: #36383f;
+  --gris: #172c4f;
+  --verde: #36b37e;
+} /* variables in spanish (for fun) */
+
+body {
+  background-color: var(--blanco);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin: 0;
+}
+
+/* ----- Elements ----- */
+
+.flex-nav-list {
+  display: none;
+}
+
+.menu {
+  display: block;
+  position: absolute;
+  right: 25px;
+  top: 25px;
+}
+
+.index-page {
+  background-image:
+    url("../img/header-illsutration-mobile.png"),
+    url("../img/Action.png");
+  background-repeat: no-repeat;
```

Comment:
- [ ] Dear @JuanDiegoGuerra, You did great adding the background image as suggested. I also notice the previous reviewer mention to look out for your margin and padding. As you can see your background image does not take the whole width of the page compared with the figma design. the background should take 100% of the view width.

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/227435721-4d411299-c97c-4f56-a731-b51f361e0d3d.png)
### Your design on 375px screen size
![image](https://user-images.githubusercontent.com/69077061/227435848-f61ec5e8-dc63-4819-ab97-8ae093bfac35.png)

Reply:
Hello again @TadesseDev . I don't know if this was a problem of my "view perspective" because I was also working with that pixel size (375) and also tested with some different phone sizes, like some Samgsung and Google devices, and they were working fine, both the background and the "action" down-arrow icon. I only had that problem with the Iphone screen size. Maybe is not optimized for all the devices but I still re-fixed it to make it good for the most common phone size and also for the specific sizes that was requierd for the project. But maybe I'm wrong, idk. I will wait patienly for another review, hope it can be you again. Thank you for your time!
I'm attaching some images that shows different size tests:

- Most common phone size: ✅
![ResponsibeMobileSize-most_common_size](https://user-images.githubusercontent.com/45726631/227446685-70421432-f9ab-4c53-bdd2-c838e3d4e4c1.JPG)

- Samsung Galaxy S8 size: ✅
![ResponsibeMobileSize-samsung_galaxy_s8](https://user-images.githubusercontent.com/45726631/227446772-8843ee7f-b314-4b17-9e0d-7955a07ec228.JPG)

- Iphone SE size: ✅
![ResponsibeMobileSize-Iphone_SE](https://user-images.githubusercontent.com/45726631/227446870-42875efb-52fd-4652-92f3-55b74976f90c.JPG)